### PR TITLE
[Industry Core KIT] doc: added new unique id push notification illustration

### PR DESCRIPTION
**Pull request by johannsvarela**
## Description
Enhancement of the existing documentation for the unique id push notifications with a new illustration to describe the functionality of the notifications in a visual way. For detailed information have a look at the changelog. 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Industry Core Kit/Software Development View/part_uniqueidpush.mdx [hunk 1]
```diff
@@ -28,7 +28,9 @@ The solution is based on notification assets in the EDC (which is the same appro
 
 > :raised_hand: It is important to understand that the receiver creates EDC asset and policies, and thus, the sender of the Unique ID push notification must check during the EDC negotiation process if the conditions the receiver offers are acceptable for the sender.
 
-Currently there are two types of Unique ID Push Notifications available: "Connect to Parent" and "Connect to Child".
+Currently there are two types of Unique ID Push Notifications available: "Connect to Parent" and "Connect to Child":
+
+![Unique ID Push Notifications](../assets/unique_id_push_notifications.svg)
```
Comment (eckardg) [inline docs-kits/kits/Industry Core Kit/Software Development View/part_uniqueidpush.mdx:33]: In the picture, please change "must" to "can"
Screenshot 2024-10-17 162306

Comment (johannsvarela) [reply docs-kits/kits/Industry Core Kit/Software Development View/part_uniqueidpush.mdx:33]: Done! 😄

**Review by eckardg — APPROVED** [review]
Thanks!

**Comment by stephanbcbauer** [comment]
@johannsvarela THX for this contribution ⇾ one question related to the picture. Is this just an exported SVG? Or could it also directly be adapted? Question ⇾ where are the sources, if someone from the open-source community want to adapt the picture.

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
As @stephanbcbauer mentioned, 
- please provide the "raw" file like the drawio
- please add a dedicated license file for the svg and raw file ([TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08#self-created-content)) -> same location and same name and append `.license` (content as mentioned in the trg with your companies that created the image OR just the contributors to the ef)

Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?

**Comment by eckardg** [comment]
@tom-rm-meyer-ISST: Question to your sidequestion: do you want to add an information where the singlelevelbom (=at customer) and where the singlelevelusage (=at supplier) is maintained? Or do you have something else in mind?

**Comment by johannsvarela** [comment]
> Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?

@stephanbcbauer @tom-rm-meyer-ISST unlike png files, svg files can be inserted and adapted directly into tools such as drwa.io. I would therefore not add an additional raw file. 

If you still insist, I'm happy to add that, but this file would never be used in KIT, but would only remain in the asset folder as an additional file.

**Comment by johannsvarela** [comment]
> As @stephanbcbauer mentioned,
> 
> * please provide the "raw" file like the drawio
> * please add a dedicated license file for the svg and raw file ([TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08#self-created-content)) -> same location and same name and append `.license` (content as mentioned in the trg with your companies that created the image OR just the contributors to the ef)
> 
> Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?

@tom-rm-meyer-ISST i'm not sure if i get your point of your sidequest? 😅

**Comment by stephanbcbauer** [comment]
> > Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?
> 
> @stephanbcbauer @tom-rm-meyer-ISST unlike png files, this svg file of the image can be inserted and adapted directly into tools such as drwa.io. I would therefore not add an additional raw file.
> 
> If you still insist, I'm happy to add that, but this file would never be used in KIT, but would only remain in the asset folder as an additional file.

Ah cool, sorry didnt test it if it is possible to open in draw.io -> it works (my concern was, not all SVG are editable in drawio

image

and its also mentioned in the svg file 

```
<!-- Do not edit this file with editors other than draw.io -->
```

File: docs-kits/kits/Industry Core Kit/assets/unique_id_push_notifications.svg.license [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- SPDX-FileCopyrightText: 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
+- SPDX-FileCopyrightText: 2023 German Edge Cloud GmbH & Co. KG
+- SPDX-FileCopyrightText: 2023 Mercedes Benz AG
+- SPDX-FileCopyrightText: 2023 Robert Bosch Manufacturing Solutions GmbH
+- SPDX-FileCopyrightText: 2023 SAP SE
+- SPDX-FileCopyrightText: 2023 Siemens AG
+- SPDX-FileCopyrightText: 2023 T-Systems International GmbH
+- SPDX-FileCopyrightText: 2023 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023 Contributors to the Eclipse Foundation
```
Comment (tom-rm-meyer-ISST) [inline docs-kits/kits/Industry Core Kit/assets/unique_id_push_notifications.svg.license:14]:
```suggestion
- SPDX-FileCopyrightText: 2024 BASF SE
- SPDX-FileCopyrightText: 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
- SPDX-FileCopyrightText: 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
- SPDX-FileCopyrightText: 2024 German Edge Cloud GmbH & Co. KG
- SPDX-FileCopyrightText: 2024 Mercedes Benz AG
- SPDX-FileCopyrightText: 2024 Robert Bosch Manufacturing Solutions GmbH
- SPDX-FileCopyrightText: 2024 SAP SE
- SPDX-FileCopyrightText: 2024 Siemens AG
- SPDX-FileCopyrightText: 2024 T-Systems International GmbH
- SPDX-FileCopyrightText: 2024 ZF Friedrichshafen AG
- SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
```

If you really created the image in 2023, ignore and resolve this comment. Else you can just commit the update to 2024.

**Comment by tom-rm-meyer-ISST** [comment]
> > As @stephanbcbauer mentioned,
> > 
> > * please provide the "raw" file like the drawio
> > * please add a dedicated license file for the svg and raw file ([TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08#self-created-content)) -> same location and same name and append `.license` (content as mentioned in the trg with your companies that created the image OR just the contributors to the ef)
> > 
> > Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?
> 
> @tom-rm-meyer-ISST i'm not sure if i get your point of your sidequest? 😅

@johannsvarela & @eckardg: my sidequestion was partially stupid as the information is partially available (after downloading the svg I can zoom :D ). When I recently read the IC KIT and this feature, I tried to think "OK which `Connect-to-X` do I need to for the bom and which for the usage direction?". For the usage you already state it in the middle of `Connect-to-Child`. To be more consistent I propose to update the upper middle text to something like "New DT has been created for a part and is available. The customer can update the singleLevelBillOfMaterial." (Note: I made it here an active tense sentence as they are easier to read concerning the "who does what" - you can also go for current passive tense as the actor is clear :) )

For me personaly I would also think about highlighting the models (e.g. make them bold).

Nevertheless, I added one comment regarding the license year (might be even wrong). Just ping me for the approval (I'll be available till monday mid of the day).

**Comment by johannsvarela** [comment]
> > > As @stephanbcbauer mentioned,
> > > 
> > > * please provide the "raw" file like the drawio
> > > * please add a dedicated license file for the svg and raw file ([TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08#self-created-content)) -> same location and same name and append `.license` (content as mentioned in the trg with your companies that created the image OR just the contributors to the ef)
> > > 
> > > Sidequestion to the image: Does it make sense to state the direction in terms of the bom (bom, usage) as we're trying to build / update that semantic model?
> > 
> > 
> > @tom-rm-meyer-ISST i'm not sure if i get your point of your sidequest? 😅
> 
> @johannsvarela & @eckardg: my sidequestion was partially stupid as the information is partially available (after downloading the svg I can zoom :D ). When I recently read the IC KIT and this feature, I tried to think "OK which `Connect-to-X` do I need to for the bom and which for the usage direction?". For the usage you already state it in the middle of `Connect-to-Child`. To be more consistent I propose to update the upper middle text to something like "New DT has been created for a part and is available. The customer can update the singleLevelBillOfMaterial." (Note: I made it here an active tense sentence as they are easier to read concerning the "who does what" - you can also go for current passive tense as the actor is clear :) )
> 
> For me personaly I would also think about highlighting the model names (e.g. make them bold).
> 
> Nevertheless, I added one comment regarding the license year (might be even wrong). Just ping me for the approval afterwards (I'll be available till monday mid of the day).

@tom-rm-meyer-ISST done as suggested! 😄

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM! Let me know, if I should trigger the merge :)

**Review by stephanbcbauer — APPROVED** [review]
LGTM, thx

**Review by eckardg — APPROVED** [review]
LGTM

**Pull request merged**